MoreLinq/From.cs

```diff
@@ -0,0 +1,126 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2017 Felipe Sateler. All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+
+namespace MoreLinq
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    partial class MoreEnumerable
+    {
+        /// <summary>
+        /// Returns a single-element sequence containing the result of invoking the function function
+        /// </summary>
+        /// <remarks>
+        /// This operator uses deferred execution and streams the results.
+        /// If the resulting sequence is enumerated multiple times, the function will be
+        /// invoked multiple times too.
+        /// </remarks>
+        /// <typeparam name="T">The type of the object returned by the function</typeparam>
+        /// <param name="function">The function to evaluate</param>
+        /// <returns>A sequence with the value created from <paramref name="function"/></returns>
```

Comment:
Instead of:

> A sequence with the value created from…

say:

> A sequence with the value resulting from invoking…

Ditto for other overloads.